testing/tests/offline-db.test.js

```diff
@@ -0,0 +1,17 @@
+const fs = require("fs");
+const path = require("path");
+const crypto = require("crypto");
+
+test("db.ts should be identical (PWA vs. Client)", () => {
+  function md5sum(path) {
+    const content = fs.readFileSync(path);
+    return crypto.createHash("md5").update(content).digest("hex");
```

Comment:
Super nit. Use `"sha256"`.
> The MD5 message-digest algorithm is a cryptographically broken but still widely used [hash function](https://en.wikipedia.org/wiki/Hash_function) producing a 128-[bit](https://en.wikipedia.org/wiki/Bit) hash value

That's wikipedia.

Reply:
Resolved via 4fa0e9fcf.